target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -293,6 +293,9 @@ ramips_setup_macs()
 	xiaomi,mi-router-4c)
 		wan_mac=$(mtd_get_mac_binary factory 0x4)
 		;;
+	xiaomi,mi-router-ra75)
+		label_mac=$(mtd_get_mac_binary factory 0x4)
+		;;
```

Comment:
Where is the `label_mac` being used?

Reply:
It isn't, as far as I can tell. It documents what is the label MAC, which is not very useful. 
Removed and verified that MACs stay the same as with stock firmware